### DISCUSSION TITLE
Deployment problem

### DISCUSSION THREAD

**Discussion by kikat234** [hmislk/hmis#6987] [Q&A]
after executing the command 'mvn package' I get a a lot of errors and warnings such as:

[WARNING] The POM for unknown.binary:org.eclipse.persistence.antlr:jar:SNAPSHOT is missing, no dependency information available
[WARNING] The POM for unknown.binary:org.eclipse.persistence.core:jar:SNAPSHOT is missing, no dependency information available
[WARNING] The POM for unknown.binary:org.eclipse.persistence.moxy:jar:SNAPSHOT is missing, no dependency information available

no matter the changes I make there is no progress can someone please help me to solve it

**Comment by ramathmanjitha27** [comment]
Hi,
Hope it was solved
